sycl/unittests/Extensions/CommandGraph/Update.cpp

```diff
@@ -7,6 +7,7 @@
 //===----------------------------------------------------------------------===//
 
 #include "Common.hpp"
+#include "sycl/exception.hpp"
```

Comment:
Is this include necessary?

Reply:
Removed. Probably added automatically by the code editor. Need to check the settings.